proposals/Simple-programs.md

```diff
@@ -47,18 +47,16 @@ compilation_unit
     ;
 ```
 
-In all but one *compilation_unit* the *statement*s must all be local function declarations. 
+Only one *compilation_unit* is allowed to have the *statement*s. 
 
 Example:
 
 ``` c#
-// File 1 - any statements
 if (args.Length == 0
```

Comment:
>args [](start = 4, length = 4)

By this spec `args` is not declared and this is an error.

Reply:
> By this spec args is not declared and this is an error.

Thanks, I will adjust the example.